File: Vibr_Inman_7_6.jl
using Plots
gr()
n = 3
while n < 24
    t = 0:(pi/n):20
    f(t) = 5*sin.(3*t)
    display(
    plot(f,t,
    seriestype = :scatter,
    title = "Response Sampling",
    xlabel = "time (s)",
    ylabel = "Transferred Output (m,F)"
    ))
    global n *= 2
    
end

File: Vibr_Inman_7_6.m
%P6
clc
clear all 
n = 3;
while n<24
    t = 0:pi/(n):20;
    f1 = 5.*sin(3.*t);
    figure
    scatter(t,f1,15,"filled","red")
    title(["Response sampling at pi/",num2str(n),"seconds"])
    xlabel("time (s)")
    ylabel("Transferred Output (m,F)")
    n=n*2;
end
